getting-started/windows-calico/rancher.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Install Calico for Windows on a Rancher Kubernetes Engine cluster
+description: Install Calico for Windows on a Rancher RKE cluster.
+canonical_url: '/getting-started/windows-calico/rancher'
+---
+
+### Big picture
+
+Install {{site.prodnameWindows}} on a Rancher Kubernetes Engine (RKE) cluster.
+
+### Value
+
+Run Linux and Windows workloads on a RKE cluster with {{site.prodname}}.
+
+### Before you begin
+
+**You will need**
+
+- An RKE cluster provisioned with {% include open-new-window.html text='no network plugin' url='https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins#disabling-deployment-of-a-network-plug-in' %}
+but which otherwise meets the {{site.prodnameWindows}} Kubernetes [cluster requirements]({{site.baseurl}}/getting-started/windows-calico/requirements). This guide was tested with RKE v1.18.9.
+- One or more Windows nodes that meet the [requirements]({{site.baseurl}}/getting-started/windows-calico/requirements).
```

Comment:
Does Rancher cluster support BGP?We should make it clear what network backend it support.

Reply:
There's nothing about RKE that prevents it. Instead of updating this line I added an example Installation cr for the Linux cluster using BGP, WDYT?